code/modules/modular_computers/file_system/programs/signaller.dm

```diff
@@ -0,0 +1,99 @@
+/datum/computer_file/program/signaller
+	filename = "signaller"
+	filedesc = "Remote Signaller"
+	category = PROGRAM_CATEGORY_MISC
+	program_icon_state = "signal"
+	extended_desc = "A small built-in frequency app that sends out signaller signals with the appropriate hardware."
+	size = 2
```

Comment:
Only size ones fit in tablets usually, you are putting these in science ones roundstart, if taken out they won't be able to put it back in.

Reply:
This is a program, the sizes are in GQ. The hardware part works fine and can be inserted